### PR TITLE
UI dark mode

### PR DESCRIPTION
**Pull request by honeybadgerofdoom**

### CODE REVIEW

File: src/iframe/css/main.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  color: var(--bootstrapButtonsHover) !important;
+  background-color: var(--bootstrapButtonsColor) !important;
+  border-color: var(--bootstrapButtonsColor) !important;
+}
+
+.btn-outline-light.focus,.btn-outline-light:focus {
+  box-shadow: var(--bootstrapButtonsFocus) !important;
+}
+
+.bs-tooltip-auto[x-placement^=top] .arrow::before,.bs-tooltip-top .arrow::before {
```
Comment (pierce-smith1) [inline src/iframe/css/main.css:33]: This is by far the most insane css selector I have ever seen

File: src/iframe/js/frame-specific/switchImageSources.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+var darkMode = true;
+function switchImageSources() {
+	var dropdownArrow = document.getElementsByClassName("dropdown-arrow");
+	var tooltipIcon = document.getElementsByClassName("tool-tip");
+	var moonIcon = document.getElementsByClassName("moon-icon");
+	if(darkMode) {
+		for(var i = 0; i < dropdownArrow.length; i++) {
```
Comment (pierce-smith1) [inline src/iframe/js/frame-specific/switchImageSources.js:7]: Prefer `let` instead of `var` and `.forEach` instead of counting for loops

**Comment by pierce-smith1** [comment]
Looks good overall, but I am running into a bug where the dropdown arrows on dark mode are rending as boxes, and the local server is spitting 404s for `/images/DropdownWhite.png` (was this supposed to be `DropdownWhite.svg`)?
image
image

**Review by pierce-smith1 — CHANGES_REQUESTED** [review]
See previous comments

File: src/iframe/css/main.css (hunk 1 above)
Comment (honeybadgerofdoom) [reply src/iframe/css/main.css:33]: haha yeah you get into some weird css when overriding Bootstrap source

File: src/iframe/js/frame-specific/switchImageSources.js (hunk 2 above)
Comment (honeybadgerofdoom) [reply src/iframe/js/frame-specific/switchImageSources.js:7]: Noted, will update

**Comment by honeybadgerofdoom** [comment]
Added the mysteriously missing image, updated loops to .forEach, updated 'var' to 'let'

**Review by pierce-smith1 — APPROVED** [review]
Works™

**Review by drg101 — APPROVED** [review]
Works for me!

**Pull request merged**